client/lib/react-test-env-setup/index.js

```diff
@@ -45,3 +45,13 @@ module.exports.cleanup = function() {
 	delete global.localStorage;
 	delete global.XMLHttpRequest;
 };
+
+module.exports.auto = function( markup, features ) {
```

Comment:
I opened PR https://github.com/Automattic/wp-calypso/pull/3857 for it. I used `setupDomEnv`. Let's discuss in related PR :)